src/Microsoft.AspNetCore.Mvc.Core/AcceptedObjectResult.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Net.Http.Headers;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    /// <summary>
+    /// An <see cref="ActionResult"/> that returns an Accepted (202) response with a Location header.
+    /// </summary>
+    public class AcceptedObjectResult : ObjectResult
```

Comment:
This naming isn't super consistent with `CreatedResult` - https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Core/CreatedResult.cs